### PR TITLE
Wps test transaction fix

### PR DESCRIPTION
**Pull request by ccmoloney**
Fix bathy layer WPS check (and any other layers that don't have a time dimension)

### CODE REVIEW

File: aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java [hunk 1]
```diff
@@ -138,17 +138,56 @@ public String getLatestTimeStep(String layer, String timeField) throws Aggregati
...
+                        "properties":{
+                            "timestep_id":819995,
+                            "collection_name":"bathy_ppb_deakin_url",
+                            "file_url":"Deakin_University/bathymetry/Victorian-coast_Bathy_10m.nc",
+                            "size":6.3451275957E10,
+                            "time":null}
+                        }],
+                    "crs":null}
+
+                    We would usually expect the 'time' property to have a va;ue - but it is possible that it is null
```
Comment (akashisama) [inline aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java]: "va;ue"

File: aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java [hunk 2]
```diff
@@ -109,7 +109,7 @@ public HttpIndexReader(String geoserver) {
     }
 
 
-    public String getLatestTimeStep(String layer, String timeField) throws AggregationException {
+    public String getLatestTimeStep(String layer, String timeField) throws AggregationException, TimeNotSupportedException {
```
Comment (akashisama) [inline aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java]: The different possible outcomes of this method are a bit confusing. It's hard to work out from reading what it means if you get a null returned vs an AggregationException vs a TimeNotSupportedException. I don't actually have any ideas for fixing that though...

File: aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java (hunk 1 above)
Comment (ccmoloney) [reply aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java]: :)

File: aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java (hunk 2 above)
Comment (ccmoloney) [reply aggregation-worker/src/main/java/au/org/emii/geoserver/client/HttpIndexReader.java]: I considered some alternatives - but this seemed like roughly the best way to do it.  I have added some comments in the code as to why it was implemented this way.

**Pull request merged**